### PR TITLE
[hotfix][doc] add section for various optimizations of reading Hive tables

### PR DESCRIPTION
**Pull request by bowenli86**
## What is the purpose of the change

add section for various optimizations of reading Hive tables

## Brief change log

add section for various optimizations of reading Hive tables

### CODE REVIEW

**Comment by bowenli86** [comment]
@xuefuz @lirui-apache @JingsongLi @zjuwangg

File: docs/dev/table/hive/read_write_hive.md [hunk 1]
```diff
@@ -137,11 +137,29 @@ Flink SQL> INSERT OVERWRITE myparttable SELECT 'Tom', 25, 'type_1', '2019-08-08'
...
 We have tested on the following of table storage formats: text, csv, SequenceFile, ORC, and Parquet.
 
-# ------ ORC Vectorized Optimization ------ 
+
+## Optimizations
+
+### Partition Pruning
+
+Flink uses partition pruning as a performance optimization to limits the number of files and partitions
+that Flink reads when querying Hive tables. When you partition data, Flink only reads a subset of the partitions in
```
Comment (JingsongLi) [inline docs/dev/table/hive/read_write_hive.md]: `When you partition data` is this a mistake gramma sentence?

**Comment by JingsongLi** [comment]
Should we add limit push down? can not add too, after all, because of our push model.

**Comment by bowenli86** [comment]
@JingsongLi thanks,  added limit push down, as we should provide information of all efforts we've done to users.

Please take another look

**Comment by JingsongLi** [comment]
Thanks @bowenli86 LGTM

**Pull request closed without merging**